Makefile

```diff
@@ -48,28 +48,28 @@ docker_compose_build:
 # dependency on generate needed to create generated file outside of docker with
 # current user owner instead of root
 docker_test: generate
-	$(DOCKER_COMPOSE) up --build --abort-on-container-exit --exit-code-from tigris_test tigris_test
+	$(DOCKER_COMPOSE) up --build tigris_test tigris_test
 
 docker_test_no_build:
-	$(DOCKER_COMPOSE) up --no-build --abort-on-container-exit --exit-code-from tigris_test tigris_test
+	$(DOCKER_COMPOSE) up --no-build tigris_test tigris_test
 
-test: clean docker_test
+test: docker_test
 
 # Use this target to run the test from inside docker container
 local_test: generate
 	go test $(TEST_PARAM) ./...
 
-run: clean generate
+run: generate
 	$(DOCKER_COMPOSE) up --build --detach tigris_server2
 
 local_run: server
-	$(DOCKER_COMPOSE) up --no-build --detach tigris_search tigris_fdb
-	fdbcli -C ./test/config/fdb.cluster --exec "configure new single memory"
+	$(DOCKER_COMPOSE) up --no-build --detach tigris_search tigris_db
+	fdbcli -C ./test/config/fdb.cluster --exec "configure new single memory" || true
```

Comment:
What happens when there is an error during `fdbcli` execution? Will this potentially hide it?

Reply:
I'll remove it. The intention was to not fail if database already exists.